pype/plugins/nuke/publish/submit_nuke_deadline.py

```diff
@@ -352,3 +356,31 @@ def expected_files(self,
         for i in range(self._frame_start, (self._frame_end + 1)):
             instance.data["expectedFiles"].append(
                 os.path.join(dir, (file % i)).replace("\\", "/"))
+
+    def get_limiting_group_filter(self):
+        """Search for limit group nodes and return group name.
+
+        Limit groups will be defined as pairs in Nuke deadline submitter
+        presents where the key will be name of limit group and value will be
+        a list of plugin's node class names. Thus, when a plugin uses more
+        than one node, these will be captured and the triggered process
+        will add the appropriate limit group to the payload jobinfo attributes.
+
+        Returning:
+            str: captured groups devided by comma and no space
+        """
+        captured_groups = []
+        for lg_name, list_node_class in self.deadline_limit_groups.items():
+            for node_class in list_node_class:
+                for node in nuke.allNodes(recurseGroups=True):
+                    # ignore all nodes not member of defined class
+                    if node.Class() not in node_class:
+                        continue
+                    # ignore all disabled nodes
+                    if node["disable"].value():
+                        continue
+                    # add group name if not already added
+                    if lg_name not in captured_groups:
+                        captured_groups.append(lg_name)
+
+        return ",".join(captured_groups)
```

Comment:
I would just return the list of limit groups and rename the method to `get_limit_groups`.

Reply:
I agree. `limiting groups` doesn't match deadline naming and methods should be called as verbs ideally